src/main/resources/db/17-create-event-table.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+    https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+    http://www.liquibase.org/xml/ns/dbchangelog
+    https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd">
+
+    <changeSet id="17-create-event-table" author="moe">
+        <createTable tableName="event">
+            <column name="event_id" type="bigint">
+                <constraints primaryKey="true" nullable="false" primaryKeyName="event_id_pk"/>
+            </column>
+            <column name="title" type="varchar(255)"/>
+            <column name="fk_image" type="bigint"/>
+            <column name="start_date_time_in_utc" type="bigint"/>
+            <column name="end_date_time_in_utc" type="bigint"/>
+            <column name="fk_event_location" type="bigint"/>
+            <column name="fk_category" type="bigint"/>
```

Comment:
add isEditable

Reply:
no, this should become an extra liquibase file